vyper/venom/passes/mem2var.py

```diff
@@ -13,6 +13,10 @@ class Mem2Var(IRPass):
     It does yet do any memory aliasing analysis, so it is conservative.
     """
 
+    # Number of memory operations over which we promote
+    # a palloca to a stack variable
+    OPS_THRESHOLD = 6
```

Comment:
not sure how i feel about this. this basically means there is something in the stack scheduler which causes stack to perform worse than memory right?

Reply:
this is here exactly because there is something wrong with the scheduler! :)